src/components/WelcomeScreen.tsx

```diff
@@ -0,0 +1,90 @@
+import { InputField } from '@components/InputField';
+import * as React from 'react'
+
+export interface Props {
+  hasAccount: boolean
+  isIncorrectPassword: boolean
+  walletName: string
+}
+
+let logo
+if (process.env.IS_WEB) {
+  logo = require('../logo2.svg')
+}
+
+export const WelcomeScreen: React.SFC<Props> = (props) => (
+  <div className="container u-padding-left-sm-2 u-padding-right-sm-2" style={{ maxWidth: '540px' }}>
+    <header className="u-margin-top-xl vertical-spaced level">
+      <figure>
+        <img src={logo} alt="Kinesis logo" />
```

Comment:
what happens here if we're in electron mode? we probably want to provide a fallback src